test/unit/mongodb/mongodb-configGroup-registry-test.js

```diff
@@ -0,0 +1,451 @@
+/*
```

Comment:
Is test/unit/mongodb/mongodb-configGroup-registry-test.js a "parallel" of some existing service-based test?

Reply:
Yes, It is parallel of test/unit/mongodb/mongodb-group-registry-test.js

Reply:
In that case, I'd suggest to add a FIXME mark at the beginning of the "parallel" file. I mean, in test/unit/mongodb/mongodb-group-registry-test.js something like this:

```
// FIXME: parallel tests in mongodb-configGroup-registry-test.js. Remove this file if at the end /iot/services API (now Deprecated) is removed
```

(And the same for all the other cases mentioned in other comments in this PR)

Reply:
Fixed in 41ee27c